tests/system-test/3-connectors/c#/TDengineDriver/TaosMultiBind.cs

```diff
@@ -581,7 +583,10 @@ public static void FreeTaosBind(TAOS_MULTI_BIND[] mBinds)
         {
             foreach (TAOS_MULTI_BIND bind in mBinds)
             {
-                Marshal.FreeHGlobal(bind.buffer);
+                if (bind.is_string)
```

Comment:
since struct need to keep as same as taosc's construct. 
you can change to  use bind.buffer_type = nchar,binary ,json, to adjust if it is a buffer for string.

Reply:
see my comment.  Thanks for your contribution.

Reply:
Got it

Reply:
@pigwing C# have been split from TDengine repo, the new repo address is https://github.com/taosdata/taos-connector-dotnet.git .  New commit should commit to this repo

Reply:
star,^_^